tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -18157,6 +18163,9 @@ TEST_F(EnergyPlusFixture, VRF_FluidTCtrl_SupplementalHtgCoilTest)
     GetVRFInput(*state);
     state->dataHVACVarRefFlow->GetVRFInputFlag = false;
 
+    state->dataHVACGlobal->OnOffFanPartLoadFraction = 1.0;
+    state->dataHVACVarRefFlow->VRF(VRFCond).VRFCondCyclingRatio = 1.0;
+
```

Comment:
These lines here and below can be removed once you revert changes to the VAV fan model.

Reply:
Indeed. I reverted the optional arg changes in the VAV fan model and removed these too.